diff -Naur -x '*~' -x '*.orig' -x '*.rej' nikto-2.1.5/nikto.conf nikto-2.1.5-fhs/nikto.conf
--- nikto-2.1.5/nikto.conf	2012-08-21 23:58:40.000000000 +0200
+++ nikto-2.1.5-fhs/nikto.conf	2012-10-16 13:12:03.539964564 +0200
@@ -61,11 +61,11 @@
 CHECKMETHODS=HEAD GET
 
 # If you want to specify the location of any of the files, specify them here
-# EXECDIR=/opt/nikto				# Location of Nikto
-# PLUGINDIR=/opt/nikto/plugins			# Location of plugin dir
-# DBDIR=/opt/nikto/databases			# Location of plugin dir
-# TEMPLATEDIR=/opt/nikto/templates		# Location of tempmlate dir
-# DOCDIR=/opt/nikto/docs			# Location of docs dir
+EXECDIR=/usr/bin
+PLUGINDIR=/usr/share/nikto/plugins
+DBDIR=/usr/share/nikto/databases
+TEMPLATEDIR=/usr/share/nikto/templates
+DOCDIR=/usr/share/doc
 
 # Default plugin macros
 @@MUTATE=dictionary;subdomain
