diff -up nikto-2.5.0/program/nikto.conf.default.1~ nikto-2.5.0/program/nikto.conf.default
--- nikto-2.5.0/program/nikto.conf.default.1~	2025-02-15 23:13:33.622901392 +0100
+++ nikto-2.5.0/program/nikto.conf.default	2025-02-15 23:14:39.576265814 +0100
@@ -56,10 +56,10 @@ CIRT=107.170.99.251
 
 # Cookies: send cookies with all requests
 # Multiple can be set by separating with a semi-colon, e.g.:
-# "cookie1=cookie value";"cookie2=cookie val" 
+# "cookie1=cookie value";"cookie2=cookie val"
 #STATIC-COOKIE="name=value";"something=nothing";
 
-# The below allows you to vary which HTTP methods are used to check whether an HTTP(s) server 
+# The below allows you to vary which HTTP methods are used to check whether an HTTP(s) server
 # is running. Some web servers, such as the autopsy web server do not implement the HEAD method
 CHECKMETHODS=GET
 
@@ -69,17 +69,22 @@ CHECKMETHODS=GET
 # DBDIR=/opt/nikto/databases            # Location of database dir
 # TEMPLATEDIR=/opt/nikto/templates      # Location of template dir
 # DOCDIR=/opt/nikto/docs                # Location of docs dir
+EXECDIR=/usr/bin
+PLUGINDIR=/usr/share/nikto/plugins
+DBDIR=/usr/share/nikto/databases
+TEMPLATEDIR=/usr/share/nikto/templates
+DOCDIR=/usr/share/doc
 
 # Default plugin macros
 # Remove plugins designed to be run standalone
 @@EXTRAS=dictionary;siebel;embedded
 @@DEFAULT=@@ALL;-@@EXTRAS;tests(report:500)
 
-# Choose SSL libs: 
-# SSLeay        - use Net::SSLeay 
-# SSL           - use Net::SSL 
-# auto          - automatically choose what's available 
-#                 (SSLeay wins if both are available) 
+# Choose SSL libs:
+# SSLeay        - use Net::SSLeay
+# SSL           - use Net::SSL
+# auto          - automatically choose what's available
+#                 (SSLeay wins if both are available)
 LW_SSL_ENGINE=auto
 
 # Number of failures before giving up
